### PR TITLE
fix(command): remove 'LISTEN' env vars

### LINKED ISSUE #201: Remove 'LISTEN' env vars
The `LISTEN` env vars (ex. `CRYOSTAT_LISTEN_HOST`) are no longer used by Cryostat and can/should therefore be removed from the Operator, since setting them no longer has any effect. These were previously used to control command channel binding on raw TCP sockets, but now that the command (notification) channel is only implemented over WebSocket, the bound interface and port always correspond to the `WEB` env var values.

### PR DESCRIPTION
**Pull request by jaadbarg**
Fixes #201

### CODE REVIEW

**Comment by ebaron** [comment]
Hey John, I think you need to rebase this PR on top of main. If you're not familiar with that workflow it's something like:
```
$ git checkout main
$ git pull https://github.com/cryostatio/cryostat-operator.git main
$ git checkout #201_ListenVar
$ git rebase main
<fix any conflicts>
$ git push -f git@github.com:jaadbarg/cryostat-operator.git
```

**Comment by ebaron** [comment]
Hi John, it appears that the unit tests in cryostat_controller_test.go need to be fixed as well. There are several assertions in there that look for the presence/absence of the `cryostat-command` service/route/ingress. Searching for `cryostat-command` should give you an idea of what you need to remove. To run the unit tests, use `make test-envtest`.

File: docs/config.md [hunk 1]
```diff
@@ -103,9 +103,14 @@ spec:
 
 ### Network Options
 When running on Kubernetes, the operator requires Ingress configurations for each of its services to make them available outside of the cluster. For a `Cryostat` object named `x`, the following Ingress configurations must be specified within the `spec.networkOptions` property:
+<<<<<<< HEAD
+- `coreConfig` exposing the service `x` on port `8181`.
+- `grafanaConfig` exposing the service `x-grafana` on port `3000`.
+=======
```
Comment (andrewazores) [inline docs/config.md:109]: Looks like some merge conflict markers made it into the commit by mistake

**Review by ebaron — COMMENTED** [review]
Hey John, I found a few more things to remove:

- `NewCommandChannelService` function: https://github.com/jaadbarg/cryostat-operator/blob/608871f1a79e55dcb3e2af36700d097e0f6c2b78/internal/controllers/common/resource_definitions/resource_definitions.go#L709
- "command" branch of `getNetworkConfig`: https://github.com/jaadbarg/cryostat-operator/blob/608871f1a79e55dcb3e2af36700d097e0f6c2b78/internal/controllers/cryostat_controller.go#L673-L674
- the `commandSVC` and `commandIng` variables: https://github.com/jaadbarg/cryostat-operator/blob/608871f1a79e55dcb3e2af36700d097e0f6c2b78/internal/test/resources.go#L1336-L1344

**Review by ebaron — APPROVED** [review]
Looks good! Thanks John!

**Pull request merged**